docs/docs/specs/v0.1/personal-space.md

```diff
@@ -0,0 +1,61 @@
+# Personal Space
+
+The user's "home" space which roams with them across all devices. This gives developers a default place to persist state for a given identity across all devices.
+
+# Requirements
+
+1. There is only one personal space per identity
+2. The contents of the personal space replicate to all devices of the identity by default
+3. Users can reset their personal space by creating a new one to replace it
+
+```ts
+
+import { Client } from '@dxos/client';
+
+const client = new Client();
+
+await client.initialize();
+
+const identity = client.halo.identity.get(); // null by default
+
+// get the personal space, creating an identity is not required
+const space = client.getPersonalSpace();
+
+// sense if a space is the personal space
+const isPersonalSpace = space.isPersonalSpace;
+
+// recreate the personal space and copy all data into the new space without history
+// (old space will disappear or archive on all devices)
+const newSpace = await client.createPersonalSpace();
```

Comment:
What happens when you join as a new device to an existing identity after creating the personal space?